tests/network-tests/src/flows/membership/validatorProfile.ts

```diff
@@ -20,6 +20,11 @@ export default async function validatorAccount({ api, query }: FlowProps): Promi
       isVerified: true,
       asWorker: 'j4VEC6FcJtBrwYQKhBAoB6Rj83jDeVua6azuHBrri1zoksBkz',
     },
+    {
+      memberId: '3',
+      isVerified: false,
+      asWorker: 'j4VEC6FcJtBrwYQKhBAoB6Rj83jDeVua6azuHBrri1zoksBkz',
+    },
```

Comment:
The problem here is that the member 3 is already not verified so this doesn't really test unverifying a member. Instead the idea would be to do something like:
```ts
const verifyValidatorFixture = new VerifyValidatorAccountFixture(api, query, VerifyValidator)
const verifyValidatorRunner = new FixtureRunner(verifyValidatorFixture)
await verifyValidatorRunner.run()
await verifyValidatorRunner.runQueryNodeChecks()

const unVerifyValidatorFixture = new VerifyValidatorAccountFixture(api, query, UnVerifyValidator)
const unVerifyValidatorRunner = new FixtureRunner(unVerifyValidatorFixture)
await unVerifyValidatorRunner.run()
await unVerifyValidatorRunner.runQueryNodeChecks()
```
Here `UnVerifyValidator` would unverify a member that was verified in `VerifyValidator`.

Reply:
Currently, there is no `unverifying` mechanism.
`Membership.leadRemark` and `Membership.workerRemark` can only `verify` validator profile.

Once, a validator profile is verified, it will automatically unverify when the validator's membership details are edited.

Reply:
Yes but this is why you added `is_verified` to `VerifyValidator` in `metadata-protobuf/proto/WorkingGroups.proto` when it's `false` the profile should get unverified.